### ISSUE TITLE
[bug] Can't update github.com/sigstore/rekor due to in-toto fork

### ISSUE THREAD

**Issue by ianlewis** [slsa-framework/slsa-github-generator#1946]
github.com/sigstore/rekor [depends on github.com/in-toto/in-toto-golang 0.7.1](https://github.com/sigstore/rekor/blob/b0b8ecbadd4a8775663e4b26aed92f098259c853/go.mod#L23) but we require a fork of -in-toto-golang based on 0.6.

We need to resolve the fork by getting the SLSA v1.0 types merged into a proper release of `in-toto-golang`.

Related to #1689

**Comment by ianlewis** [comment]
Related to #1909

**Comment by ianlewis** [comment]
Blocked on https://github.com/in-toto/in-toto-golang/issues/197

**Comment by ianlewis** [comment]
This was fixed.
